[14.0] [MIG] website_sale_stock_list_preview: Migration to 14.0

CarlosRoca13 opened the pull request:
@Tecnativa 

please @sergio-teruel @Tardo review this

CarlosRoca13 committed:
[13.0][ADD] website_sale_stock_list_preview: warehouse added to context

CarlosRoca13 committed:
[IMP] website_sale_stock_list_preview: hide the add to cart button when no stock available

CarlosRoca13 committed:
[FIX] website_sale_stock_list_preview: Added parameter virtual_available_formated to show the quantity available

claudiagn committed:
Added translation using Weblate (Spanish)

CarlosRoca13 committed:
[IMP] website_sale_stock_list_preview: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_stock_list_preview: Migration to 14.0

LudLaf commented on website_sale_stock_list_preview/readme/DESCRIPTION.rst:
```diff
@@ -0,0 +1,2 @@
+This module extends the functionality of e-Commerce to give
+the buyer the information about the stock on the preview of the products.
```
```suggestion
This module extends e-Commerce's functionality to give the buyer information about what's in stock when previewing the products
```

LudLaf commented on website_sale_stock_list_preview/readme/USAGE.rst:
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+* Edit some products on the option Availability of e-commerce
```
```suggestion
* Edit the 'Availability' option under 'eCommerce' tab in your selected product.
```

LudLaf commented on website_sale_stock_list_preview/readme/USAGE.rst:
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+* Edit some products on the option Availability of e-commerce
+* Go to the shop on the website and active Add to Cart view on Customize menu.
```
```suggestion
* Go to the website's shop and activate the 'Add to Cart' view under 'Customize' menu.
```

LudLaf commented on website_sale_stock_list_preview/readme/USAGE.rst:
```diff
@@ -0,0 +1,5 @@
+
+* Edit some products on the option Availability of e-commerce
+* Go to the shop on the website and active Add to Cart view on Customize menu.
+* You can see on your products the results of the options that you had choose.
```
```suggestion

You can see, on your products, the options you have chosen.
```

LudLaf reviewed (changes requested):
Take a look at the README

CarlosRoca13 commented:
Changes done @LudLaf

pedrobaeza commented:
/ocabot migration website_sale_stock_list_preview

pilarvargas-tecnativa reviewed (approved):
locally tested :+1:

pedrobaeza reviewed (approved):
/ocabot merge nobump

pedrobaeza committed:
[FIX] website_sale_paymen_term_acquirer: Avoid unlink error in tests

odoo.exceptions.UserError: You cannot delete the payment acquirer Partner's PT; archive it instead.

pedrobaeza commented:
/ocabot merge nobump
